package.json

```diff
@@ -85,6 +86,7 @@
     "react": "^16.14.0 || ^17.0.0"
   },
   "dependencies": {
-    "@stitches/react": "^1.2.5"
```

Comment:
I see we moved this from dependencies to devDependencies. If we are going to do this, then we need to add `@stitches/react` to `peerDependencies` which would constitute a major/breaking change.